configure.ac

```diff
@@ -38,6 +38,13 @@ AC_ARG_ENABLE(
 	[enable_distonly="no"]
 )
 
+AC_ARG_ENABLE(
+	[password-save],
+	[AS_HELP_STRING([--disable-password-save], [disable password save support @<:@default=yes@:>@])],
+	[enable_password_save="yes"],
+	[enable_password_save="no"]
+)
```

Comment:
On Mon, Aug 1, 2016 at 2:19 PM, Ilya Shipitsin <notifications@github.com>
wrote:

> In configure.ac
> <https://github.com/OpenVPN/openvpn-gui/pull/60#discussion_r73026272>:
>
> > @@ -38,6 +38,13 @@ AC_ARG_ENABLE(
> >  	[enable_distonly="no"]
> >  )
> >
> > +AC_ARG_ENABLE(
> > +	[password-save],
> > +	[AS_HELP_STRING([--disable-password-save], [disable password save support @<:@default=yes@:>@])],
> > +	[enable_password_save="yes"],
> > +	[enable_password_save="no"]
> > +)
>
> ok, what's wrong ?
>
Looking at your failed travis build log I see that configure is called as

./configure --prefix=/ --libdir=/lib --host=$CHOST
--build=x86_64-pc-linux-gnu --program-prefix='' && make

No --disable-password-change passed? I'm not familiar with travis, but I
guess $CONFIG  (=--disable-password-change, in the last two builds) has to
somehow get into the command line?

Selva


This will set enable_password_save to no if no options are specified and
and to yes if either disable-password-save or enable-password-save is
specified. We want none of that. So write it as


AC_ARG_ENABLE([password-save],[AS_HELP_STRING([--disable-password-save],
              [disable password save support @<:@default=yes@:>@])],
              [],
              [enable_password_save="no"]
)



commit: chipitsine@49f8f4e
> <https://github.com/chipitsine/openvpn-gui/commit/49f8f4edfe0f2096684182854368819eaf33d0d3>
> build: https://travis-ci.org/chipitsine/openvpn-gui/builds/148951122
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenVPN/openvpn-gui/pull/60/files/551efac831f1b4c73614dbb9dd580043efb96af6..208a383d29c16bc402c382b472a7288cfc0fab3e#r73026272>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADzAT5oMmeyqNx-tYjTpTIWot065ai-Qks5qbjiwgaJpZM4I8BrK>
> .
>